run_websocket_server.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 from torchvision import datasets
 from torchvision import transforms
-
+from syft.frameworks.torch.federated import utils
```

Comment:
Q: Shouldn't here be fl?

Reply:
I was running on an older version of syft-proto. It has to be fl according to new version.

Reply:
resolved.